### PR TITLE
Add controller for user role assigment

### PR DESCRIPTION
**Pull request by FirestarJes**
Adds a controller to handle assign user roles to a user.

### CODE REVIEW

File: apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public class MarketParticipantUserRoleAssignmentController : MarketParticipantControllerBase
+    {
+        private readonly IMarketParticipantUserRoleAssignmentClient _client;
+
+        public MarketParticipantUserRoleAssignmentController(IMarketParticipantUserRoleAssignmentClient client)
+        {
+            _client = client;
+        }
+
+        [HttpGet]
```
[the remark this answers is not in the document]
Comment (FirestarJes) [reply apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs:35]: No, it should be a POST

[the remark this answers is not in the document]
Comment (FirestarJes) [reply apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs]: fixed

File: apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        private readonly IMarketParticipantUserRoleAssignmentClient _client;
+
+        public MarketParticipantUserRoleAssignmentController(IMarketParticipantUserRoleAssignmentClient client)
+        {
+            _client = client;
+        }
+
+        [HttpPut]
+        [Route("users/roles")]
+        public Task GetAsync(Guid userId, UpdateUserRoleAssignmentsDto updateUserRoleAssignmentsDto)
```
Comment (xnetsrak) [inline apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs:37]: Name could be changed as well, GetAsync is not great here.

Comment (FirestarJes) [reply apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs:37]: OK

Comment (FirestarJes) [reply apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleAssignmentController.cs]: Fixed

**Review by defectiveAi — APPROVED** [review]

**Review by mimse — APPROVED** [review]

**Pull request merged**